### ISSUE TITLE
openapi version 3.0

### ISSUE THREAD

**Issue by adamdottv** [rhinobase/hono-openapi#18]
Is it possible to output other versions of the OpenAPI spec (3.0.0 for instance)? I'm running into issues generating off of the v3.1 spec and wondering if this was on the roadmap (or supported already).

**Comment by MathurAditya724** [comment]
Yes, support for OpenAPI 3.0.0 is in progress! Could you share the specific issues you’re encountering with the 3.1 spec?

**Comment by adamdottv** [comment]
The tool I'm using to generate API docs (Mintlify) fails to parse the 3.1 specs, but handles 3.0 specs. I've also reached out to them and am awaiting reply.

**Comment by MathurAditya724** [comment]
Hey👋🏼 any update from the Mintilify team regarding this?
